azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -357,4 +353,20 @@ private JobRunner createJobRunner(final int execId, final String name, final int
     return runner;
   }
 
+  private void assertThreadIsWaiting(final Thread thread) throws Exception {
+    TestUtils.await().until(
+        () -> thread.getState() == TIMED_WAITING || thread.getState() == WAITING);
+  }
+
+  private void assertThreadIsNotAlive(final Thread thread) throws Exception {
+    thread.join(9_000L);
```

Comment:
Curious what does `9_000L` mean? it doesn't look like a number.

Reply:
Heh, this is because I first had 10_000L, where the separation is good for readability. With 9000L it's not necessary. It's a perfectly valid syntax [since Java 7](http://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html), though.